### PR TITLE
Chopsticks Update for v0.6.4

### PR DESCRIPTION
**Pull request by jboetticher**
### Description

Chopsticks has had some recent updates, which means some of the commands don't work like they used to. `dry-run` still doesn't work with Moonbeam though.

I have also simplified & clarified sections of the docs.

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [X] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/build/substrate-api/chopsticks.md [hunk 1]
```diff
@@ -77,22 +68,30 @@ The simplest way to fork Moonbeam is through the previously introduced configura
...
+|  Method   |                                             Command                                             |
+|:------------:|:-----------------------------------------------------------------------------------------------:|
+|    Name   |        `npx @acala-network/chopsticks --config=moonbeam`                        |
+|    URL    | `npx @acala-network/chopsticks --config=https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/moonbeam.yml`                  |
+| File Path | Requires a local download of the [Moonbeam configuration file](https://github.com/AcalaNetwork/chopsticks/blob/master/configs/moonbeam.yml){target=_blank}.<br>`npx @acala-network/chopsticks --config=configs/moonbeam.yml`                   |
+
 A configuration file is not necessary, however. There are additional commands and flags to configure the environment completely in the command line.  
 
-The `npx @acala-network/chopsticks dev` command forks a chain, and includes following flags:  
+The `npx @acala-network/chopsticks` command forks a chain, and includes following flags:
```
[the remark this answers is not in the document]
Comment (eshaben) [reply builders/build/substrate-api/chopsticks.md:94]: it feels like this should be in the prior section, but then i'm not so sure that the headers align

File: builders/build/substrate-api/chopsticks.md [hunk 2]
```diff
@@ -112,7 +111,7 @@ The `npx @acala-network/chopsticks dev` command forks a chain, and includes foll
 
 When running a fork, by default it will be accessible at `ws://localhost:8000`. You will be able to interact with the parachain via libraries such as [Polkadot.js](https://github.com/polkadot-js/common){target=_blank} and its [user interface, Polkadot.js Apps](https://github.com/polkadot-js/apps){target=_blank}.
```
Comment (eshaben) [inline builders/build/substrate-api/chopsticks.md:112]: break ws://localhost:8000 to a new line and wrap it in three backticks so its easy to copy

File: builders/build/substrate-api/chopsticks.md [hunk 3]
```diff
@@ -77,22 +68,30 @@ The simplest way to fork Moonbeam is through the previously introduced configura
...
 === "Moonriver"
     ```
-    npx @acala-network/chopsticks dev --config=https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/moonriver.yml
+    npx @acala-network/chopsticks --config=https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/moonriver.yml
     ```
 
 === "Moonbase Alpha"
     ```
-    npx @acala-network/chopsticks dev --config=https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/moonbase-alpha.yml
+    npx @acala-network/chopsticks --config=https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/moonbase-alpha.yml
```
[the remark this answers is not in the document]
Comment (eshaben) [reply builders/build/substrate-api/chopsticks.md:81]: can you also break apart the rest of the commands in this file please?

File: builders/build/substrate-api/chopsticks.md [hunk 4]
```diff
@@ -153,23 +152,26 @@ npx @acala-network/chopsticks run-block --endpoint wss://wss.api.moonbeam.networ
 To test out XCM messages between networks, you can fork multiple parachains and a relay chain locally. For example, the following will fork Moonriver, Karura, and Kusama given that you've downloaded the [config folder from the GitHub repository](https://github.com/AcalaNetwork/chopsticks/tree/master/configs){target=_blank}:  
 
 ```
-npx @acala-network/chopsticks xcm --relaychain=configs/kusama.yml --parachain=configs/moonriver.yml --parachain=configs/karura.yml
+npx @acala-network/chopsticks xcm --r=kusama.yml --p=moonriver.yml --p=karura.yml
```
Comment (eshaben) [inline builders/build/substrate-api/chopsticks.md:155]:
```suggestion
npx @acala-network/chopsticks xcm \
--r=kusama.yml \
--p=moonriver.yml \
--p=karura.yml
```

File: builders/build/substrate-api/chopsticks.md [hunk 5]
```diff
@@ -153,23 +152,26 @@ npx @acala-network/chopsticks run-block --endpoint wss://wss.api.moonbeam.networ
 To test out XCM messages between networks, you can fork multiple parachains and a relay chain locally. For example, the following will fork Moonriver, Karura, and Kusama given that you've downloaded the [config folder from the GitHub repository](https://github.com/AcalaNetwork/chopsticks/tree/master/configs){target=_blank}:
```
Comment (eshaben) [inline builders/build/substrate-api/chopsticks.md:152]: could probably add a bit more clarity here and say that you've downloaded the entire `configs` directory or at the very least the config for Moonriver, Karura, and Kusama.

File: builders/build/substrate-api/chopsticks.md (hunk 4 above)
Comment (eshaben) [reply builders/build/substrate-api/chopsticks.md:155]: maybe add a note here stating that the chain name and raw github url can also be used

File: builders/build/substrate-api/chopsticks.md [hunk 6]
```diff
@@ -191,12 +193,20 @@ Each method can be invoked by connecting to the websocket (`ws://localhost:8000`
...
 - **`values` Object** - a JSON object resembling the path to a storage value, similar to what you would retrieve via Polkadot.js  
-- **`blockHash` string** - optional, the blockhash at which the storage value is changed
-- **`date` string** - a Date string (compatible with the JavaScript Date library) that will change the time stamp from which the next blocks being created will be at. All future blocks will be sequentially after that point in time
-- **`hashOrNumber` number | string** - if found, the chain head will be set to the block with the block number or block hash of this value
+- **`blockHash` string** - optional, the blockhash at which the storage value is changed  
+- **`date` Date** - a Date string (compatible with the JavaScript Date library) that will change the time stamp from which the next blocks being created will be at. All future blocks will be sequentially after that point in time  
+- **`hashOrNumber` number | string** - if found, the chain head will be set to the block with the block number or block hash of this value
```
Comment (eshaben) [inline builders/build/substrate-api/chopsticks.md:210]: this seems like it should be above the actual json rpc request

**Review by eshaben — CHANGES_REQUESTED** [review]
everything worked as expected!

the bottom table has parameters wrapped in back ticks, but the rest of the tables in the guide do not. can you please add back ticks to anything that is a part of a command or code-related?

File: builders/build/substrate-api/chopsticks.md [hunk 7]
```diff
@@ -77,22 +68,30 @@ The simplest way to fork Moonbeam is through the previously introduced configura
...
     ```
-    npx @acala-network/chopsticks dev --config=https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/moonbase-alpha.yml
+    npx @acala-network/chopsticks --config=https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/moonbase-alpha.yml
     ```
 
+When providing the config flag, you can use their raw URL, a path to the file, or simply use the chain's name. For example, the following commands all use Moonbeam's configuration in the same way:  
+
+|  Method   |                                             Command                                             |
+|:------------:|:-----------------------------------------------------------------------------------------------:|
+|    Name   |        `npx @acala-network/chopsticks --config=moonbeam`                        |
```
[the remark this answers is not in the document]
Comment (jboetticher) [reply builders/build/substrate-api/chopsticks.md:91]: This makes the width of the table larger than its container, causing the entire table to become a scrollview. Is that ok?

**Review by eshaben — CHANGES_REQUESTED** [review]
I played around with the table for the name, URL, and local path and ended up switching it to tabs and using a note as I think thats what looks best.

As for the Forking section it just seems redundant to the config section. So maybe just merge those two sections as a "Configure and Fork Moonbeam with Chopsticks" section or something

**Comment by jboetticher** [comment]
I tried my best to combine them. Also included a left-over quickstart section... not sure if it fits or not

File: builders/build/substrate-api/chopsticks.md [hunk 8]
```diff
@@ -13,106 +13,181 @@ description: Learn the basics of how to use Chopsticks to replay blocks, dissect
...
+
+A configuration file is not necessary, however. All of the settings (except `genesis` and `timestamp`) can also be passed as flags to configure the environment completely in the command line. For example, the following command forks Moonbase Alpha at block 100.
+
+```
+npx @acala-network/chopsticks --endpoint {{ networks.moonbase.rpc_url }} --block 100
+```
+
+### Quickstart {: #quickstart }
+
+The simplest way to fork Moonbeam is through the configuration files that are stored in the chopsticks GitHub repository:
```
Comment (eshaben) [inline builders/build/substrate-api/chopsticks.md:160]:
```suggestion
The simplest way to fork Moonbeam is through the configuration files that are stored in the Chopsticks GitHub repository:  
```

**Review by eshaben — APPROVED** [review]
yeah I mean it does feel a little out of place, but i think it's fine in case someone just wants the command to get started without getting into the configuration details

**Pull request merged**